src/Components/Web.Extensions/src/InputFile/IBrowserFile.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Components.Web.Extensions
+{
+    /// <summary>
+    /// Represents the data of a file selected from an <see cref="InputFile"/> component.
+    /// </summary>
+    public interface IBrowserFile
```

Comment:
Do we need an API review?

Reply:
This has been scheduled. We plan on getting this in and following up with a deep dive \ api review next week.